templates/coderd.yaml

```diff
@@ -40,6 +40,8 @@ spec:
     spec:
       securityContext: {{ toYaml .Values.coderd.podSecurityContext | nindent 8 }}
       restartPolicy: Always
+      imagePullSecrets:
+      - name: {{ toYaml .Values.coderd.imagePullSecret }}
```

Comment:
Should wrap with an `if` so we don't include this is `coderd.imagePullSecret` is empty.